via/views/view_pdf.py

```diff
@@ -67,6 +67,18 @@ def proxy_google_drive_file(request):
     return _iter_pdf_response(request.response, content_iterable)
 
 
+@view_config(route_name="proxy_jstor_pdf", decorator=(has_secure_url_token,))
+def proxy_jstor_pdf(context, request):
+    jstor_api = request.find_service(JSTORAPI)
+    if not jstor_api.enabled:
+        raise HTTPUnauthorized()
```

Comment:
The UI for this is pretty bad. The request to the pdf fails but pdf.js doesn't show too much indication of what's going on. 

I think for us, for debugging etc, it's enough.

Reply:
Is there a way of adding an error message here that will be exposed at least in the HTTP response if the developer is looking at that?